packages/fx-core/tests/component/driver/aad/update.test.ts

```diff
@@ -478,6 +478,41 @@ describe("aadAppUpdate", async () => {
       );
   });
 
+  it("should throw permissionErrorCode when update failed with CannotDeleteOrUpdateEnabledEntitlement", async () => {
+    sinon.stub(AadAppClient.prototype, "updateAadApp").rejects({
+      isAxiosError: true,
+      response: {
+        status: 400,
+        data: {
+          error: {
+            code: "CannotDeleteOrUpdateEnabledEntitlement",
+            message:
```

Comment:
The message is not the one returned by AAD. So the test case cannot detect the issue that `error.aad.manifest.AADPermissionIssues` is not appended to AAD's error message.

Reply:
removed